### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by undermoodzyx** [philc/vimium#2067]
Hello, I found the *vimium* is a very useful extension.And I want to share it to others.
Is it possible that you could allow me to translate the **README.md** into Chinese?
Thank you!

**Comment by smblott-github** [comment]
@undermoodzyx... That sounds like an idea for the Wiki.

**Comment by undermoodzyx** [comment]
@smblott-github 
Yeah, but I'm just ready to do part of it, such as the **basic using**.
In addition, I want to share it in other platform like *sina blog*.
Would you mind?

**Comment by smblott-github** [comment]
> Would you mind?

That would be great.

**Comment by philc** [comment]
Closing -- unfortunately I don't have the bandwidth to maintain translated documentation.
